pkg/requestor/requestor.go

```diff
@@ -42,6 +42,7 @@ func NewRequestor(amqpURI, sendqName, exchangeName, topic, repoURL, kind, target
 	return r
 }
 
+//initQueues initializes rabbitmq queues for PR, Branch or Tag. Returns error if fails
 func (r *Requestor) initQueus() error {
```

Comment:
There is a typo in the function name

Reply:
I will address this typo in my next commit, Thanks for letting me know.